pegasus/sites.v3/code.org/views/csf_languages.haml

```diff
@@ -0,0 +1,5 @@
+%h2 CS Fundamentals (Courses 1-4) is supported in the following languages
```

Comment:
Can you link `CS Fundamentals` to https://code.org/educate/curriculum/elementary-school. Use a relative path:

`%a{href: "/educate/curriculum/elementary-school"} CS Fundamentals`

Do the same for the other tutorials, using `CDO.studio_url('/s/tutorial')`:

- Designer: /s/minecraft
- Adventurer: /s/mc
- Frozen: /s/frozen
- Star Wars: /s/starwarsblocks